src/mlpack/bindings/markdown/binding_info.cpp

```diff
@@ -3,6 +3,11 @@
  * @author Ryan Curtin
  *
  * Implementation of BindingInfo functions.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
```

Comment:
The license is missing in a bunch of file in the markdown folder, maybe this is intentional @rcurtin?

Reply:
Hehe, nope.  I just failed, that's all. :)  @atulim if you want, since you're already doing it in this PR, if you want to add this license text to all the files that are missing it, it would be great.  If not I can handle it, just let me know.

Reply:
> Hehe, nope. I just failed, that's all. :) @atulim if you want, since you're already doing it in this PR, if you want to add this license text to all the files that are missing it, it would be great. If not I can handle it, just let me know.

@rcurtin  Since I am currently reading all the files related to binding, it would be convenient for me to check for documentation. I will correct documentation missings, if any. :)

Reply:
The license issue ended up getting fixed before 3.1.0, by the way, so no need to handle other files.